From 602dd847f17ae800854ce06133b395a53c42b4c4 Mon Sep 17 00:00:00 2001
From: Alan Tull <r80115@freescale.com>
Date: Fri, 3 Feb 2012 04:16:29 -0600
Subject: [PATCH 0945/2463] ENGR00173810 clean up rej file

Makefile.rej got committed from doing cherry-pick for kernel
upgrade from 2.6.38 to 3.0.15

Signed-off-by: Alan Tull <r80115@freescale.com>
---
 sound/soc/imx/Makefile.rej |   15 ---------------
 1 files changed, 0 insertions(+), 15 deletions(-)

diff --git a/sound/soc/imx/Makefile.rej b/sound/soc/imx/Makefile.rej
deleted file mode 100644
index 7ff4796..0000000
--- a/sound/soc/imx/Makefile.rej
+++ /dev/null
@@ -1,15 +0,0 @@
---- sound/soc/imx/Makefile
-+++ sound/soc/imx/Makefile
-@@ -14,10 +14,12 @@
- snd-soc-phycore-ac97-objs := phycore-ac97.o
- snd-soc-wm1133-ev1-objs := wm1133-ev1.o
- snd-soc-imx-sgtl5000-objs := imx-sgtl5000.o
-+snd-soc-imx-cs42888-objs := imx-cs42888.o
- snd-soc-imx-spdif-objs := imx-spdif.o
- 
- obj-$(CONFIG_SND_SOC_EUKREA_TLV320) += snd-soc-eukrea-tlv320.o
- obj-$(CONFIG_SND_SOC_PHYCORE_AC97) += snd-soc-phycore-ac97.o
- obj-$(CONFIG_SND_MXC_SOC_WM1133_EV1) += snd-soc-wm1133-ev1.o
- obj-$(CONFIG_SND_SOC_IMX_SGTL5000) += snd-soc-imx-sgtl5000.o
-+obj-$(CONFIG_SND_SOC_IMX_CS42888) += snd-soc-imx-cs42888.o
- obj-$(CONFIG_SND_SOC_IMX_SPDIF) += snd-soc-imx-spdif.o
-- 
1.7.7.4

